developer/js/build.sh

```diff
@@ -16,38 +16,97 @@ build () {
   npm run build || fail "Could not build top-level JavaScript file."
 }
 
+set_version () {
+  local version=$1
+  npm --no-git-tag-version --allow-same-version version "$version" || fail "Could not set package version to $version."
```

Comment:
Please write a comment explaining why we used `--no-git-tag-version`, especially with relation to how our CI currently works!